ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-nightly.yaml

```diff
@@ -1955,13 +1955,21 @@ tests:
   cron: 53 6 26 * *
   steps:
     cluster_profile: ibmcloud-qe
+    env:
+      IBMCLOUD_COMPUTE_ENCRYPTION_KEY: "true"
+      IBMCLOUD_CONTROL_PLANE_ENCRYPTION_KEY: "true"
+      IBMCLOUD_DEFAULT_MACHINE_ENCRYPTION_KEY: "false"
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-ibmcloud-ipi-byo-kms
 - as: ibmcloud-ipi-byo-kms-f28-destructive
   cron: 31 2 23 * *
   steps:
     cluster_profile: ibmcloud-qe
+    env:
+      IBMCLOUD_COMPUTE_ENCRYPTION_KEY: "false"
+      IBMCLOUD_CONTROL_PLANE_ENCRYPTION_KEY: "false"
+      IBMCLOUD_DEFAULT_MACHINE_ENCRYPTION_KEY: "true"
```

Comment:
Those are the same with the default value, so it can be skipped, to make the jobs simpler.